azerothcore/azerothcore-wotlk
# DB/Spells: Fix various spells

## Changes Proposed:
-  Fix Various spells.


## Issues Addressed:
- Closes 



## SOURCE:



## Tests Performed:

- Compile on Windows without errors.


## How to Test the Changes:

``ALERT``: In the first case, you must try two powers, these should never fail when thrown on an enemy.
- Log in with a warrior, cast several times ``[Intercept]`` and ``[Shattering Throw]`` on an enemy and should never fail any of these 2 spells.
- Log in with a warrior, cast``[Intercept]`` several times  an enemy and one of the attempts must be critical.
- Log in with a dk, apply to your character ``[Glyph of Chains]``, cast  ``[Chains of Ice]`` several times  an enemy and one of the attempts must be critical.

``NOTE``: The main objective of this RP is that the powers ``[Shattering Throw and Intercept]`` cannot fail in addition to that ``[Intercept and Glyph of Chains]`` can be critical hits. In addition to all this, I have also readjusted the type of damage that each power must perform (for example, in the case of the warrior it would be physical damage)
## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
